platform/qt/src/qmapboxgl.cpp

```diff
@@ -147,7 +141,7 @@ std::unique_ptr<const mbgl::SpriteImage> toSpriteImage(const QImage &sprite) {
 /*!
     \enum QMapboxGLSettings::GLContextMode
 
-    This enum set the expectations towards the GL state.
+    This enum sets the expectations towards the GL state.
```

Comment:
Can we assume users know what an `enum` is? I assume yes, but just wanna make sure.